src/components/Hero/ProfileHero/Profile.js

```diff
@@ -153,20 +153,35 @@ function Profile({ classes, dominion, geoId, history, ...props }) {
           );
         }
         let { squareKms } = data.geo;
-        squareKms = parseFloat(squareKms);
+        const squareKmsFloat = parseFloat(squareKms);
         if (!Number.isNaN(squareKms)) {
-          if (squareKms < 1.0) {
-            squareKms = squareKms.toFixed(3);
+          if (squareKmsFloat < 1.0) {
+            const numberFormatter = new Intl.NumberFormat('en-IN', {
+              minimumFractionDigits: 3,
+              maximumFractionDigits: 3
+            });
+            squareKms = numberFormatter.format(squareKmsFloat);
           } else {
-            squareKms = squareKms.toFixed(1);
+            const numberFormatter = new Intl.NumberFormat('en-IN', {
+              minimumFractionDigits: 1,
+              maximumFractionDigits: 1
+            });
+            squareKms = numberFormatter.format(squareKmsFloat);
           }
         }
 
         let population;
         let populationDensity;
         if (totalPopulation) {
-          population = totalPopulation.toFixed(0);
-          populationDensity = (totalPopulation / squareKms).toFixed(2);
+          let numberFormatter = new Intl.NumberFormat('en-IN');
+          population = numberFormatter.format(totalPopulation.toFixed(0));
+          numberFormatter = new Intl.NumberFormat('en-IN', {
```

Comment:
This number format can be reused I think

Reply:
There is no way to reset the options @kilemensi

Reply:
I meant you could reuse the one declared on L167 above. It's exactly as this one but it's no biggie.